Catappult/appcoins-sdk
# [feature/APT-3725] Server side payflow

**What does this PR do?**

  Adds backend payflow logic to dynamically change the flow to show the user, and its priority in case multiple flows are available.
  Currently bind services will have priority compared to alternative flows that wont include having an external app handling the bind action.
  Adds Gameshub as a bind service and removes the Cafe bazaar service handle.

**Database changed?**

   No

**Where should the reviewer start?**

- PayflowManager.kt
- WalletUtils.java
- RepositoryServiceConnection.java
- AppcoinsBillingStubHelper.java
- AppcoinsBillingWrapper.java

**How should this be manually tested?**

  Have two bind services installed (wallet and GamesHub) and check the priority to see which service will be called.

**What are the relevant tickets?**

  Tickets related to this pull-request: [APT-3725](https://aptoide.atlassian.net/browse/APT-3725)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.) 


**Code Review Checklist**
